pkg/operator/ceph/version/version.go

```diff
@@ -224,9 +226,9 @@ func IsInferior(a, b CephVersion) bool {
 func ValidateCephVersionsBetweenLocalAndExternalClusters(localVersion, externalVersion CephVersion) error {
 	logger.Debugf("local version is %s, external version is %s", localVersion.String(), externalVersion.String())
 
-	// We only support Nautilus or newer
-	if !externalVersion.IsAtLeastNautilus() {
-		return fmt.Errorf("unsupported ceph version %s, need at least nautilus, delete your cluster CR and create a new one with a correct ceph version", externalVersion.String())
+	// We only support Luminous or newer
+	if !externalVersion.IsAtLeast(Luminous) {
```

Comment:
Luminous is no longer supported for Rook. Do we still need this check?

Reply:
It's no longer supported for Rook when it configures a storage cluster but we could certainly support it for external cluster configuration.